Gemeinsame Unterverzeichnisse: phonon-4.5.1.orig/cmake und phonon-4.5.1/cmake.
Gemeinsame Unterverzeichnisse: phonon-4.5.1.orig/demos und phonon-4.5.1/demos.
Gemeinsame Unterverzeichnisse: phonon-4.5.1.orig/designer und phonon-4.5.1/designer.
Gemeinsame Unterverzeichnisse: phonon-4.5.1.orig/includes und phonon-4.5.1/includes.
Gemeinsame Unterverzeichnisse: phonon-4.5.1.orig/phonon und phonon-4.5.1/phonon.
diff -u phonon-4.5.1.orig/qt_phonon.pri phonon-4.5.1/qt_phonon.pri
--- phonon-4.5.1.orig/qt_phonon.pri	2011-10-21 21:03:49.000000000 +0200
+++ phonon-4.5.1/qt_phonon.pri	2011-10-31 00:16:58.081648105 +0100
@@ -3,6 +3,5 @@
 # the next build of Qt. So pretend we're not there in this case.
 # This file is loaded by qt_config.prf, before .qmake.cache has been loaded.
 # Consequently, we have to do some stunts to get values out of the cache.
-if(!exists($$_QMAKE_CACHE_)| \
-   !contains($$list($$fromfile($$_QMAKE_CACHE_, CONFIG)), QTDIR_build)): \
+if(!exists($$_QMAKE_CACHE_)|!contains($$list($$fromfile($$_QMAKE_CACHE_, CONFIG)), QTDIR_build)): \
     QT_CONFIG += phonon
